packages/extension-api/src/extension-api.d.ts

```diff
@@ -2378,6 +2482,58 @@ declare module '@podman-desktop/api' {
      * A test to perform to check that the container is healthy.
      */
     HealthCheck?: HealthConfig;
+
+    /**
+     *  Default: ```false```
+     *
+     * Command is already escaped (Windows only)
+     */
+    ArgsEscaped?: boolean;
+
+    /**
+     * An object mapping mount point paths inside the container to empty objects.
+     */
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    Volumes?: { [volume: string]: {} };
```

Comment:
can we remove all the ban-types ?

Reply:
I don't know which type as the Docker reference does not detail it

https://docs.docker.com/engine/api/v1.44/#tag/Container/operation/ContainerCreate